modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -120,9 +120,12 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
   def load(q: engine.EventQueue, seqId: SPObservationID): Task[SeqexecFailure \/ Unit] = {
     val t = EitherT(for {
       odbSeq <- Task(odbProxy.read(seqId))
-    } yield odbSeq.flatMap(s => translator.sequence(translatorSettings)(seqId, s))
+    } yield odbSeq.map(s => translator.sequence(translatorSettings)(seqId, s))
     )
-    val u = t.flatMapF(x => q.enqueueOne(Event.load(seqId.stringValue(), x)).map(_.right))
+    val u = t.flatMapF{
```

Comment:
This produces a warning
```
[warn] /Users/cquiroz/Projects/ocs3/modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala:125: match may not be exhaustive.
[warn] It would fail on the following input: (Nil, None)
[warn]     val u = t.flatMapF{
```